packages/@react-spectrum/button/stories/Button.stories.tsx

```diff
@@ -89,6 +90,10 @@ storiesOf('Button', module)
   .add(
     'element: a, rel: \'noopener noreferrer\'',
     () => render({elementType: 'a', href: '//example.com', rel: 'noopener noreferrer', variant: 'primary'})
+  )
+  .add(
+    'user-select test',
+    () => <Example />
```

Comment:
you can add a description to these now https://github.com/adobe/react-spectrum/blob/1e635d5e6540927a001f552194c30d8b3b48d8e8/packages/%40react-aria/selection/stories/useSelectableList.stories.tsx#L122